### DISCUSSION TITLE
Auto install dotnet sdk.

### DISCUSSION THREAD

**Discussion by Doprez** [stride3d/stride#1642] [Contributors]
Is there a way we could securely install the Dotnet SDK when running stride?

Currently the Gamestudio just shows an error message and expects the user to know what is needed. Can we use something like Winget or Chocolatey to install the SDK when this error is thrown?

https://github.com/stride3d/stride/issues/1612
https://github.com/stride3d/stride/pull/1641

**Comment by manio143** [comment]
I don't think we should be installing anything silently. If we detect .NET 6 SDK to be absent we can show a dialog to the user asking if he wants to install it and if they say yes, then send them to the download page for it.

**Comment by NicusorN5** [comment]
It would be even better if .NET 6 SDK would get installed during the setup, I guess.
